Parsers/ASimWebSession/Parsers/ASimWebSessionPaloAltoNetworks.yaml

```diff
@@ -0,0 +1,98 @@
+      [   1, "Informational" 
+          , 2, "Low" 
+          , 3, "Medium"
+          , 4, "Medium" 
+          , 5, "High"
+      ];
+      CommonSecurityLog
+      | where DeviceVendor == "Palo Alto Networks"
+          and DeviceProduct == "PAN-OS"
+          and Activity == "THREAT"
+          and DeviceEventClassID == "url"
+      | parse AdditionalExtensions with URL ";PanOSActionFlags=" *
+      | extend 
+          Url                         = coalesce(RequestURL,URL)
+      | lookup EventLookup on DeviceAction
+      | lookup SeverityLookup on LogSeverity
+      | project Url, EventResult, HttpStatusCode, Computer, DeviceCustomString4, DeviceCustomString2, DeviceAction, _ItemId, LogSeverity,DeviceVendor, DeviceVersion, DeviceInboundInterface, DestinationIP, DestinationPort, SourceIP, SourcePort,SourceUserName,DestinationUserName, RequestMethod, RequestContext, ThreatConfidence, Protocol, DeviceCustomNumber1, TimeGenerated
+      | project-rename 
+          Dvc                         = Computer
+          , DvcZone                   = DeviceCustomString4
+          , UrlCategory               = DeviceCustomString2
+          , EventResultDetails        = DeviceAction
+          , EventUid                  = _ItemId
+          , EventOriginalSeverity     = LogSeverity
+          , EventProductVersion       = DeviceVersion
+          , DvcInterface              = DeviceInboundInterface
+          , DstIpAddr                 = DestinationIP
+          , DstPortNumber             = DestinationPort
+          , SrcIpAddr                 = SourceIP
+          , SrcPortNumber             = SourcePort
+          , SrcUsername               = SourceUserName
+          , DstUsername               = DestinationUserName
+      | extend 
+          EventType                   = "HTTPsession"
+          , EventSchema               = "WebSession"
+          , EventSchemaVersion        = "0.2.5"
+          , EventProduct              = "Palo Alto Networks"
+          , EventVendor               = "PAN-OS"
+          , EventStartTime            = TimeGenerated
+          , EventEndTime              = TimeGenerated
+          , HttpRequestMethod         = toupper(RequestMethod)
+          , HttpContentFormat         = RequestContext
+          , Dst                       = DstIpAddr
+          , Src                       = SrcIpAddr
+          , NetworkProtocolVersion    = case(
+              DstIpAddr contains "."  , "IPv4"
+              , DstIpAddr contains ":", "IPv6"
+              , "")
+          , IpAddr                    = SrcIpAddr
+          , ThreatConfidence          = toint(ThreatConfidence)
+          , NetworkProtocol           = toupper(Protocol)
+          , NetworkSessionId          = tostring(DeviceCustomNumber1)
+          , User                      = SrcUsername
+      | extend DvcOs                  = EventVendor
+          , DvcOsVersion              = EventProductVersion
```

Comment:
DvcOs and DvcOsVersion are for real OSs, so Windows, MacOS, Android. Let's not put PanOS as an OS there, as it is anyway reported in EventVendor/Product/ProductVersion.

Reply:
OK